src/views/questionView.js

```diff
@@ -26,3 +26,23 @@ export const createQuestionElement = (questionNumber, question) => {
 
   return element;
 };
+
+export const initScore = () => {
+  const score = document.body.appendChild(document.createElement('div'));
```

Comment:
We should avoid accessing the document directly in the view directly. Instead this function should return the htmlelement, and the calling code (in questionPage.js) should add it to the DOM.